### PR TITLE
Update dependencies for 3.7.8 release

### PR DESCRIPTION
**Pull request by Lex-2008**
huh?

### CODE REVIEW

File: build-scripts/package-msi [hunk 1]
```diff
@@ -77,12 +77,21 @@ package()
...
+    then
+      convert=$(printf '%d' "'$alphabet")
+      REVISION=$(echo $REVISION|sed -e "s/$alphabet/$convert/");
+    else
+      break
+    fi
+  done
+
+  # ensure all revision components are 4 digits max (MSI allows each part to be up to 65534, but we limit it to 9999)
+  REVISION=$(echo $REVISION| tr '.' '\n' | sed 's/\(....\).*/\1/' | tr '\n' '.' | sed 's/\.$//')
```
Comment (vpodzime) [inline build-scripts/package-msi]: Shouldn't the ``\(....\)`` be ``\(\d\d\d\d\)``, i.e. digits?

Comment (Lex-2008) [reply build-scripts/package-msi]: it should be all digits at this point. Probably I can add `s/[^0-9]//g`, i.e. "delete all non-digits" somewhere here.

**Comment by Lex-2008** [comment]
Ready to be reviewed! https://ci.cfengine.com/job/pr-pipeline/399/

File: deps-packaging/pcre/cfbuild-pcre.spec [hunk 2]
```diff
@@ -26,7 +28,11 @@ if [ -z $MAKE ]; then
 fi
 
 $MAKE
-if ! [ $SYS = "AIX" ]; then
+# $SYS env variable disappeared somewhere,
+# so we'll use Jenkins-provided $label var,
+# which is PACKAGES_ppc64_aix_53 on AIX.
+# if ! [ "z$SYS" = "zAIX" ]; then
+if ! expr "z$label" : '.*_aix_'; then
```
Comment (vpodzime) [inline deps-packaging/pcre/cfbuild-pcre.spec]: Please revert/fix this on all these branches.

**Pull request merged**